lib/capybara/selector.rb

```diff
@@ -19,156 +19,156 @@
+#       * :disabled (Boolean) - Match disabled field?
+#       * :multiple (Boolean) - Match field that accepts multiple values
+#       * :style (String, Regexp, Hash)
 #
 # * **:label** - Find label elements
 #   * Locator: Match id or text contents
 #   * Filters:
-#     * :for (Element, String, Regexp) - The element or id of the element associated with the label
+#       * :for (Element, String, Regexp) - The element or id of the element associated with the label
 #
 # * **:table** - Find table elements
 #   * Locator: id or caption text of table
 #   * Filters:
-#     * :id (String, Regexp, XPath::Expression) - Match id attribute of table
-#     * :caption (String) - Match text of associated caption
-#     * :class ((String, Array<String>, Regexp, XPath::Expression) - Matches the class(es) provided
-#     * :style (String, Regexp, Hash)
-#     * :with_rows (Array<Array<String>>, Array<Hash<String, String>>) - Partial match `<td>` data - visibility of `<td>` elements is not considered
-#     * :rows (Array<Array<String>>) - Match all `<td>`s - visibility of `<td>` elements is not considered
-#     * :with_cols (Array<Array<String>>, Array<Hash<String, String>>) - Partial match `<td>` data - visibility of `<td>` elements is not considered
-#     * :cols (Array<Array<String>>) - Match all `<td>`s - visibility of `<td>` elements is not considered
+#       * :id (String, Regexp, XPath::Expression) - Match id attribute of table
+#       * :caption (String) - Match text of associated caption
+#       * :class ((String, Array<String>, Regexp, XPath::Expression) - Matches the class(es) provided
+#       * :style (String, Regexp, Hash)
+#       * :with_rows (Array<Array<String>>, Array<Hash<String, String>>) - Partial match `<td>` data - visibility of `<td>` elements is not considered
+#       * :rows (Array<Array<String>>) - Match all `<td>`s - visibility of `<td>` elements is not considered
+#       * :with_cols (Array<Array<String>>, Array<Hash<String, String>>) - Partial match `<td>` data - visibility of `<td>` elements is not considered
+#       * :cols (Array<Array<String>>) - Match all `<td>`s - visibility of `<td>` elements is not considered
 #
 # * **:table_row** - Find table row
 #   * Locator: Array<String>, Hash<String,String> table row `<td>` contents - visibility of `<td>` elements is not considered
 #
 # * **:frame** - Find frame/iframe elements
 #   * Locator: Match id or name
 #   * Filters:
-#     * :id (String, Regexp, XPath::Expression) - Match id attribute
-#     * :name (String) - Match name attribute
-#     * :class (String, Array<String>, Regexp, XPath::Expression) - Matches the class(es) provided
-#     * :style (String, Regexp, Hash)
+#       * :id (String, Regexp, XPath::Expression) - Match id attribute
+#       * :name (String) - Match name attribute
+#       * :class (String, Array<String>, Regexp, XPath::Expression) - Matches the class(es) provided
+#       * :style (String, Regexp, Hash)
```

Comment:
[note] It's necessary to deepen 3rd level indentation, please see an example below (which was one before the update).

![image](https://user-images.githubusercontent.com/473530/58443875-e6f87780-812f-11e9-9d51-a385d8d37b4d.png)